gluetool/help.py

```diff
@@ -161,7 +164,13 @@ def rst_to_text(text):
     :returns: plain text representation of ``text``.
     """
 
-    return docutils.core.publish_string(text, writer=sphinx.writers.text.TextWriter(DummyTextBuilder))
+    text = docutils.core.publish_string(text, writer=sphinx.writers.text.TextWriter(DummyTextBuilder))
+
+    # in Python3 the publish_string returns bytes, convert it to string
+    if isinstance(text, bytes):
```

Comment:
I'd rather check its callers, I see no reason why `rst_to_text` should be called with bytes in Python 3. It's used to process strings, mostly docstrings, those should be `str` :/

Reply:
yes, seems so:
```
In [9]: type(docutils.core.publish_string("test"))
Out[9]: bytes
```